### DISCUSSION TITLE
BetaBindPort = 3001

### DISCUSSION THREAD

**Discussion by liang-hiwin** [AdguardTeam/AdGuardHome#4833] [Q&A]
Excuse me, when will the UI of port 3001 for this development test be released?

**Comment by liang-hiwin** [comment]
![EO04N889U`MKZ)@$X7E6A%G](https://user-images.githubusercontent.com/17548936/184470533-8e20eb63-7642-46fd-b5ed-a68554273c58.png)

**Comment by ainar-g** [comment]
No exact timeframes, unfortunately.
